packages/lib-classifier/src/store/subjects/Subject/FreehandLineReductions/FreehandLineReductions.js

```diff
@@ -15,29 +15,31 @@ const FreehandLineReductions = types
   }))
   .views(self => {
     return {
-      findCurrentTaskMark({ stepKey, tasks, frame }) {
+      findCurrentTaskMarks({ stepKey, tasks, frame }) {
         if (stepKey === undefined || tasks === undefined || frame === undefined) {
           return
         }
 
-        let caesarMark;
+        let caesarMarks = [];
 
         self.reductions.forEach(reduction => {
-          let { data } = reduction
-          let task = tasks[data.taskIndex]
-          let tool = task.tools[data.toolIndex]
-
-          if (data.frame === frame
-            && data.stepKey === stepKey
-            && data.taskKey === task.taskKey
-            && data.taskType === task.type
-            && data.toolType === tool.type
-          ) {
-            caesarMark = data
-          }
+          let { data } = reduction.data   // Caesar expects data attribute to be an object
```

Comment:
Should this be: 
```suggestion
          let { data } = reduction   // Caesar expects data attribute to be an object
```

Reply:
This is a silly thing where it's reduction.data.data = [] (it's a comment that Caesar cannot have reduction.data = []). Might be better to rename to reduction.data.marks or something similar.